lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -730,28 +732,35 @@ def customize_vm(self, vm_obj):
         # Network settings
         adaptermaps = []
         for network in self.params['networks']:
-            if 'ip' in network and 'netmask' in network:
-                guest_map = vim.vm.customization.AdapterMapping()
-                guest_map.adapter = vim.vm.customization.IPSettings()
+            guest_map = vim.vm.customization.AdapterMapping()
+            guest_map.adapter = vim.vm.customization.IPSettings()
```

Comment:
I see. I'll move the `guest_map.adapter = vim.vm.customization.IPSettings()` to be inside the `if 'ip' in network:` and `elif 'dhcp' in network:` conditions. Then I'll move all of the other `.adapter` property changes inside a block that checks for the `adapter` property.